docs/src/data/globalnav.ts

```diff
@@ -52,10 +54,10 @@ export const LEFT_NAV_LINKS = [
     url: 'https://ui.docs.amplify.aws/',
     order: 4,
   },
-  {
-    type: 'DEFAULT',
-    label: 'Contribute',
-    url: 'https://www.contributetoamplify.com/',
-    order: 5,
-  },
+  // {
```

Comment:
Do we need to comment when to uncomment this and ln31-38?

Reply:
Done